libs/ddhub-client-gateway-message-broker/src/lib/services/ddhub-messages.service.ts

```diff
@@ -38,6 +38,47 @@ export class DdhubMessagesService extends DdhubBaseService {
     );
   }
 
+  @Span('ddhub_mb_messagesAckBy')
+  public async messagesAckBy(
+    messageIds: string[],
+    clientId?: string
+  ): Promise<string[]> {
+    const requestBody = {
+      messageIds,
+      clientId
+    };
+
+    try {
+      const result = await this.request<string[]>(
+        () =>
+          this.httpService.post('/messages/ack', requestBody, {
```

Comment:
if this is message acknowledgement I would send & store a date of message ACK - even for debugging purposes

Reply:
yes